go.mod

```diff
@@ -66,6 +66,8 @@ require (
 	gopkg.in/yaml.v2 v2.3.0
 )
 
+require github.com/google/go-cmp v0.5.8
```

Comment:
seems somewhat unlikely, I don't think you added a go-cmp import anywhere.  try tidying again?

Reply:
huh.  yeah that is in use.  no idea why it was indirect before 🤔

LGTM